site/faq.md

```diff
@@ -0,0 +1,81 @@
+
+Use `weave ps` to see which address was allocated to a container on a Weave network.  
+
+See [Address Allocation](/site/features/address-allocation.md).
+
+
+###Q: How do I expose one of my containers to the outside world?
+
+Exposing a container to the outside world is described in [Exposing Services to the Outside](/site/using-weave/service-export.md).
+
+
+###Q: My dockerized app needs to check the request of a remote IP. Can I manually change the IP of a container?
+
+
+You can manually change the IP of a container using [Classless Inter-Domain Routing or CIDR notation](https://en.wikipedia.org/wiki/Classless_Inter-Domain_Routing). 
+
+For more information, refer to [Manually Specifying the IP Address of a Container](/site/using-weave/manual-ip-address.md). 
+
+
+###Q:Can I connect my existing 'legacy' network with a Weave container network?
+
+Yes you can. 
+
+Let's say you have a Weave network that runs on hosts A, B, C. and an additional host, that we'll call P, where neither Weave nor Docker are running.  You want to connect from a process running on host P to a specific container running on host B on the Weave network. 
+
+Since the Weave network is completely separate from any network that P is connected to, you cannot connect the container using the container's IP address. Instead you will have to expose a port on the container on host B. 
+
+You can read about how to expose a port in [Service Exporting](/site/using-weave/service-export.md)
+
+
+###Q: Why am I seeing the same IP address assigned to two different containers on different hosts?
+
+Under normal circumstances, this should not happen, but it can occur if  `weave forget` and `weave rmpeer` was run on more than one host. 
+
+You cannot call `weave rmpeer` on more than one host. The address space, which was owned by the stale peer cannot be left dangling, and as a result it gets reassigned. In this instance, the address is reassigned to the peer on which `weave rmpeer` was run. Therefore, if you run `weave forget` and then `weave rmpeer` on more than one host at a time, it results in duplicate IPs on more than one host.
+
+Once the peers detect the inconsistency, they log the error and drop the connection that supplied the inconsistent data. The rest of the peers will carry on with their view of the world, but the network will not function correctly.
+
+Some peers may be able to communicate their claim to the others before they run `rmpeer` (i.e. it's a race), so what you can expect is a few cliques of peers that are still talking to each other, but repeatedly dropping attempted connections with peers in other cliques.
+
+
+###Q: What is the best practise for resetting a node that goes out of service?
+
+When a node goes out of service, the best option is to call `weave rmpee` on one host and then `weave forget` on all the other hosts.
+
+This will provide enough time for Weave to re-establish its peer connections across the mesh network. 
```

Comment:
Maybe this line has got moved here by mistake?  There isn't anything to re-establish if all that happened is you removed a node.